_topic_map.yml

```diff
@@ -52,6 +52,8 @@ Topics:
   File: rosa-setting-up-environment
 - Name: Creating a ROSA cluster
   File: rosa-creating-cluster
+- Name: Creating an AWS PrivateLink cluster
```

Comment:
Should this be `Creating an AWS PrivateLink ROSA cluster`?

Reply:
@jonquilwilliams We talked about this previously, but can you take a look at this more finalized version and see if it makes sense to change this heading to  Creating an AWS PrivateLink ROSA cluster?

Reply:
@kmccarron-rh  @dustman9000  I feel that that's a strange amalgamation that flies into the face of branding issues. And in parts of the documentation, it's referred to as a "Multi-AZ PrivateLink cluster". But I get what you're driving at, so I'd be ok with this instead: `Creating an AWS PrivateLink cluster with ROSA` or `Creating ROSA clusters with AWS PrivateLink`. Though I think that makes for a very long chapter heading in the TOC.

Reply:
Actually, scratch the last suggestion of `Creating ROSA clusters with AWS PrivateLink` because all of the other headings in that section call it a PrivateLink cluster, not a ROSA cluster. So I'd only be ok with `Creating an AWS PrivateLink cluster with ROSA` (or "on ROSA")

Reply:
Thanks for the reply! I could only see some of your other responses earlier- I think my vpn might have dropped between changes.  How about we go with Creating an AWS PrivateLink cluster on ROSA unless someone has an objection?

Reply:
Sounds good to me.